exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -189,6 +189,15 @@ drill.exec: {
   debug: {
     return_error_for_failure_in_cancelled_fragments: false
   }
+  udf: {
+    retry.times: 5,
+    directory: {
+      base: "/tmp",
```

Comment:
Would have to be "/user/drill/tmp". And, of course, based on earlier comments, should be "${drill.dfs.root}/tmp".